CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# v2.0.0
+# v2.0.0-rc.2
+WIP
+
+# v2.0.0-rc.1
 This is our first **release candidate** for v2.0.0 of Reaction. Please check it out and let us know what works and what doesn't for you.
```

Comment:
*second?

Reply:
It's the WIP changelog here, but yeah, the second.